core/drivers/scmi-msg/entry.c

```diff
@@ -39,16 +40,14 @@ void scmi_process_message(struct scmi_msg *msg)
 	case SCMI_PROTOCOL_ID_BASE:
 		handler = scmi_msg_get_base_handler(msg);
 		break;
-#ifdef CFG_SCMI_MSG_CLOCK
 	case SCMI_PROTOCOL_ID_CLOCK:
-		handler = scmi_msg_get_clock_handler(msg);
+		if (IS_ENABLED(CFG_SCMI_MSG_CLOCK))
+			handler = scmi_msg_get_clock_handler(msg);
```

Comment:
This part is worng. Misses stubs as reported in another P-R https://github.com/OP-TEE/optee_os/pull/4121#discussion_r500238348.

Reply:
fixed